### PR TITLE
Fix for Events and Reminders to Work on Load & Round Calc Fix

### PR DESCRIPTION
**Pull request by JustinFreitas**
Fix for the reminders and events working correctly on load.  They need to be opened and closed to get them in memory so their data is available.  No flicker can be observed.

### CODE REVIEW

File: desktop/desktop_panels.xml [hunk 1]
```diff
@@ -56,6 +56,13 @@
 						onSourceChanged();
 						local nDateinMinutes = TimeManager.getCurrentDateinMinutes();
 						DB.setValue(TimeManager.CAL_DATEINMIN, "number", nDateinMinutes);
+
+						Interface.openWindow("timedevents", "DB");
+						Interface.toggleWindow("timedevents", "DB");
+						Interface.openWindow("timedreminders", "DB");
+						Interface.toggleWindow("timedreminders", "DB");
+						Interface.openWindow("travelmanager", "DB");
+						Interface.toggleWindow("travelmanager", "DB");
```
Comment (JustinFreitas) [inline desktop/desktop_panels.xml:67]: Opening and closing the window on Clock Adjuster window initialization allows for events and reminders to trigger successfully on load w/out having to open the windows manually first.

File: desktop/desktop_panels.xml [hunk 2]
```diff
@@ -437,7 +444,7 @@
 						local nDay = window.addday.getValue() or 0;
 						local nMonth = window.addmonth.getValue() or 0;
 						local nYear = window.addyear.getValue() or 0;
-						local nRounds = (nHour * 600) + (nMinute * 10) + (nDay * 14400) + (nDay * 403200);
+						local nRounds = (nHour * 600) + (nMinute * 10) + (nDay * 14400) + (nMonth * 403200);
```
Comment (JustinFreitas) [inline desktop/desktop_panels.xml:449]: Bug fix for repeated nDay... second one should be nMonth.

File: scripts/longtermeffects.lua [hunk 3]
```diff
@@ -11,19 +11,19 @@ function advanceRoundsOnTimeChanged(nRounds)
 			for _,nodeEffect in pairs(DB.getChildren(nodeCT, 'effects')) do
 				local nActive = DB.getValue(nodeEffect, 'isactive', 0);
 				if nActive ~= 0 then
-					local nodeCT = nodeEffect.getChild('...');
+					local nodeActor = nodeEffect.getChild('...');
 					local nDuration = DB.getValue(nodeEffect, 'duration');
 					local bHasDuration = (nDuration and nDuration ~= 0);
 					if bHasDuration and (nDuration <= nRounds) then
-						EffectManager.expireEffect(nodeCT, nodeEffect);
+						EffectManager.expireEffect(nodeActor, nodeEffect);
```
Comment (JustinFreitas) [inline scripts/longtermeffects.lua:18]: Linter fix for variable shadowing of nodeCT from outer loop.

File: scripts/longtermeffects.lua [hunk 4]
```diff
@@ -212,9 +212,9 @@ local function nextRound_new(nRounds, bTimeChanged)
 		end
 		-- end checking for necessity of full processing of rounds
 
-		for i = 1,#aEntries do
-			CombatManager.onTurnStartEvent(aEntries[i]);
-			CombatManager.onTurnEndEvent(aEntries[i]);
+		for j = 1,#aEntries do
+			CombatManager.onTurnStartEvent(aEntries[j]);
+			CombatManager.onTurnEndEvent(aEntries[j]);
```
Comment (JustinFreitas) [inline scripts/longtermeffects.lua:217]: Linter fix for variable shadowing of for loop.  Using j.

File: scripts/longtermeffects.lua (hunk 3 above)
Comment (bmos) [reply scripts/longtermeffects.lua:18]: technically not a problem, but definitely better to use a different variable name as you do here.
thanks!

File: desktop/desktop_panels.xml (hunk 1 above)
Comment (bmos) [reply desktop/desktop_panels.xml:67]: what a simple solution! funny that the window's onClose function doesn't remove the handler and cause reminders to stop working again.

**Review by bmos — APPROVED** [review]

**Pull request merged**